src/app/view/tools/insertTextTool.js

```diff
@@ -39,7 +39,8 @@ export default class InsertTextTool extends MouseTool {
 
     createNewFeature(point) {
```

Comment:
Since we are no longer getting the text from registry, we need to modify the insert text tool to have a class property as `text`

Reply:
I've added 'text' as well as getter & setter functions for use in other files. However, I did not add it to the constructor. Should I set it to a random string?

Reply:
It needs to be a part of the constructor so that so that when we initialize in the insert text tool, we can pass the text from the UI dialog into it

Reply:
Makes sense I saw how you used the setter